.happy/terraform/modules/sfn_config/nextstrain-dbg.wdl

```diff
@@ -68,30 +68,31 @@ task nextstrain_workflow {
     )
 
     # set up ncov
-    mkdir -p /ncov/my_profiles/aspen /ncov/results
-    cd /ncov
-    git init
-    git fetch --depth 1 git://github.com/nextstrain/ncov.git
-    git checkout FETCH_HEAD
-    ncov_git_rev=$(git rev-parse HEAD)
+    mkdir -p ncov/my_profiles/aspen ncov/results
+    (cd ncov &&
+     git init &&
+     git fetch --depth 1 git://github.com/nextstrain/ncov.git &&
+     git checkout FETCH_HEAD
+    )
+    ncov_git_rev=$(cd ncov && git rev-parse HEAD)
 
-    cp /usr/src/app/aspen/workflows/nextstrain_run/nextstrain_profile/* /ncov/my_profiles/aspen/
+    cp /usr/src/app/aspen/workflows/nextstrain_run/nextstrain_profile/* ncov/my_profiles/aspen/
```

Comment:
i think this still needs to be` /ncov/my_profiles/aspen/`?

Reply:
Thanks for taking a look Phoenix!! This PR aims to put `ncov` into the current working directory instead of at the root `/ncov`, therefore all the disappearing `/` throughout.

Reply:
ohhh and why did we end up doing that?

Reply:
i'm just curious and i wasn't at the initial part of the discussion

Reply:
i think it just makes things easier to navigate :D